### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.Services.AsynchronousOperationException
  id: AsynchronousOperationException
  children:
  - System.IdentityModel.Services.AsynchronousOperationException.#ctor
  - System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Exception)
  - System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String)
  - System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: AsynchronousOperationException
  nameWithType: AsynchronousOperationException
  fullName: System.IdentityModel.Services.AsynchronousOperationException
  type: Class
  summary: "Eccezione generata quando si verifica un errore durante un&quot;operazione asincrona."
  syntax:
    content: 'public class AsynchronousOperationException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.Services.AsynchronousOperationException
  langs:
  - csharp
  name: AsynchronousOperationException()
  nameWithType: AsynchronousOperationException.AsynchronousOperationException()
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException()
  type: Constructor
  assemblies:
  - System.IdentityModel.Services
  namespace: System.IdentityModel.Services
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.IdentityModel.Services.AsynchronousOperationException&quot;> </xref> classe."
  remarks: "Questo costruttore inizializza la <xref:SystemException.Message>proprietà della nuova istanza di un messaggio fornito dal sistema che descrive l&quot;errore.</xref:SystemException.Message> Questo messaggio prende in considerazione le impostazioni cultura correnti.       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per la nuova istanza.      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException>| `null`.|   | <xref:System.Exception.Message>| La stringa di messaggio di errore localizzato. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: public AsynchronousOperationException ();
    parameters: []
  overload: System.IdentityModel.Services.AsynchronousOperationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Exception)
  id: '#ctor(System.Exception)'
  parent: System.IdentityModel.Services.AsynchronousOperationException
  langs:
  - csharp
  name: AsynchronousOperationException(Exception)
  nameWithType: AsynchronousOperationException.AsynchronousOperationException(Exception)
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException(Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel.Services
  namespace: System.IdentityModel.Services
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.IdentityModel.Services.AsynchronousOperationException&quot;> </xref> classe con un messaggio di errore fornito dal sistema e un riferimento all&quot;eccezione interna che è la causa dell&quot;eccezione corrente."
  remarks: "Un&quot;eccezione generata come risultato diretto di un&quot;eccezione precedente deve includere un riferimento all&quot;eccezione precedente nella <xref:System.Exception.InnerException>proprietà.</xref:System.Exception.InnerException> Il <xref:System.Exception.InnerException>proprietà restituisce lo stesso valore passato al costruttore.</xref:System.Exception.InnerException>       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per la nuova istanza.      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException>| Il riferimento all&quot;eccezione interna. |   | <xref:System.Exception.Message>| Un messaggio fornito dal sistema che descrive l&quot;errore.</xref:System.Exception.Message> </xref:System.Exception.InnerException> Questo messaggio prende in considerazione le impostazioni cultura correnti. |"
  syntax:
    content: public AsynchronousOperationException (Exception innerException);
    parameters:
    - id: innerException
      type: System.Exception
      description: "Il <xref:System.Exception>che è la causa dell&quot;eccezione corrente.</xref:System.Exception> Se il `innerException` parametro non è <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, l&quot;eccezione corrente viene generata in un <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blocco che gestisce l&quot;eccezione interna."
  overload: System.IdentityModel.Services.AsynchronousOperationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.Services.AsynchronousOperationException
  langs:
  - csharp
  name: AsynchronousOperationException(String)
  nameWithType: AsynchronousOperationException.AsynchronousOperationException(String)
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel.Services
  namespace: System.IdentityModel.Services
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.IdentityModel.Services.AsynchronousOperationException&quot;> </xref> classe con un messaggio di errore specificato."
  remarks: "Questo costruttore inizializza la <xref:System.Exception.Message>proprietà della nuova eccezione utilizzando il `message` parametro.</xref:System.Exception.Message> Il contenuto di `message` deve essere facilmente comprensibile. Il chiamante del costruttore è necessario per garantire che la stringa sia stata localizzata per la lingua corrente.       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per la nuova istanza.      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException>| `null`.|   | <xref:System.Exception.Message>| La stringa di messaggio di errore. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: public AsynchronousOperationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio di errore che spiega il motivo dell&quot;eccezione."
  overload: System.IdentityModel.Services.AsynchronousOperationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.Services.AsynchronousOperationException
  langs:
  - csharp
  name: AsynchronousOperationException(SerializationInfo,StreamingContext)
  nameWithType: AsynchronousOperationException.AsynchronousOperationException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel.Services
  namespace: System.IdentityModel.Services
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.IdentityModel.Services.AsynchronousOperationException&quot;> </xref> classe con dati serializzati."
  remarks: "Questo costruttore viene chiamato durante la deserializzazione per ricostituire l&quot;oggetto eccezione trasmesso su un flusso."
  syntax:
    content: protected AsynchronousOperationException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Oggetto <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> oggetto che contiene i dati oggetto serializzati."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Oggetto <xref:System.Runtime.Serialization.StreamingContext>oggetto che contiene le informazioni contestuali sull&quot;origine o destinazione.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.Services.AsynchronousOperationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.Services.AsynchronousOperationException
  langs:
  - csharp
  name: AsynchronousOperationException(String,Exception)
  nameWithType: AsynchronousOperationException.AsynchronousOperationException(String,Exception)
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel.Services
  namespace: System.IdentityModel.Services
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.IdentityModel.Services.AsynchronousOperationException&quot;> </xref> classe con un messaggio di errore specificato e un riferimento all&quot;eccezione interna che è la causa dell&quot;eccezione corrente."
  remarks: "Un&quot;eccezione generata come risultato diretto di un&quot;eccezione precedente deve includere un riferimento all&quot;eccezione precedente nella <xref:System.Exception.InnerException>proprietà.</xref:System.Exception.InnerException> Il <xref:System.Exception.InnerException>proprietà restituisce lo stesso valore passato al costruttore.</xref:System.Exception.InnerException>       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per la nuova istanza.      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException>| Il riferimento all&quot;eccezione interna. |   | <xref:System.Exception.Message>| La stringa di messaggio di errore. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: public AsynchronousOperationException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio di errore che spiega il motivo dell&quot;eccezione."
    - id: innerException
      type: System.Exception
      description: "Il <xref:System.Exception>che è la causa dell&quot;eccezione corrente.</xref:System.Exception> Se il `innerException` parametro non è <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, l&quot;eccezione corrente viene generata in un <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blocco che gestisce l&quot;eccezione interna."
  overload: System.IdentityModel.Services.AsynchronousOperationException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor
  parent: System.IdentityModel.Services.AsynchronousOperationException
  isExternal: false
  name: AsynchronousOperationException()
  nameWithType: AsynchronousOperationException.AsynchronousOperationException()
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException()
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Exception)
  parent: System.IdentityModel.Services.AsynchronousOperationException
  isExternal: false
  name: AsynchronousOperationException(Exception)
  nameWithType: AsynchronousOperationException.AsynchronousOperationException(Exception)
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException(Exception)
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String)
  parent: System.IdentityModel.Services.AsynchronousOperationException
  isExternal: false
  name: AsynchronousOperationException(String)
  nameWithType: AsynchronousOperationException.AsynchronousOperationException(String)
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.Services.AsynchronousOperationException
  isExternal: false
  name: AsynchronousOperationException(SerializationInfo,StreamingContext)
  nameWithType: AsynchronousOperationException.AsynchronousOperationException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.Services.AsynchronousOperationException
  isExternal: false
  name: AsynchronousOperationException(String,Exception)
  nameWithType: AsynchronousOperationException.AsynchronousOperationException(String,Exception)
  fullName: System.IdentityModel.Services.AsynchronousOperationException.AsynchronousOperationException(String,Exception)
- uid: System.IdentityModel.Services.AsynchronousOperationException.#ctor*
  parent: System.IdentityModel.Services.AsynchronousOperationException
  isExternal: false
  name: AsynchronousOperationException
  nameWithType: AsynchronousOperationException.AsynchronousOperationException
